ckanext/qa/i18n/en_AU/LC_MESSAGES/ckanext-qa.po

```diff
@@ -0,0 +1,250 @@
+# This file is distributed under the same license as the ckanext-qa project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: ckanext-qa 2.0\n"
+"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
+"POT-Creation-Date: 2020-07-09 01:18+0000\n"
+"PO-Revision-Date: 2020-07-01 16:37+0000\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: en_AU\n"
+"Language-Team: en_AU <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.3.4\n"
+
+#: ckanext/qa/reports.py:83 ckanext/qa/reports.py:89 ckanext/qa/reports.py:150
+msgid "total_stars"
+msgstr "total_ticks"
+
+#: ckanext/qa/reports.py:84 ckanext/qa/reports.py:90 ckanext/qa/reports.py:151
+msgid "average_stars"
+msgstr "average_ticks"
+
+#: ckanext/qa/reports.py:165
+msgid "openness"
+msgstr "data_usability_rating"
+
+#: ckanext/qa/reports.py:166
+msgid "Openness (Five Stars)"
+msgstr "Data usability rating"
+
+#: ckanext/qa/reports.py:167
+msgid ""
+"Datasets graded on Tim Berners Lees' Five Stars of Openness - openly "
+"licensed, openly accessible, structured, open format, URIs for entities, "
+"linked."
+msgstr ""
+"Datasets rated according to rules similar to those used by the "
+"Australian Government's Linked Data Rating derived from "
+"(but different to) Tim Berners-Lee's Five Stars of Openness."
+
+#: ckanext/qa/tasks.py:251
+msgid "Could not understand the file format, therefore score is 1."
+msgstr ""
+
+#: ckanext/qa/tasks.py:261
+#, python-format
+msgid "Unknown error: %s"
+msgstr ""
+
+#: ckanext/qa/tasks.py:275
+msgid "License not open"
+msgstr ""
+
+#: ckanext/qa/tasks.py:302
+msgid "File could not be downloaded."
+msgstr ""
+
+#: ckanext/qa/tasks.py:303 ckanext/qa/tasks.py:378 ckanext/qa/tasks.py:383
+#: ckanext/qa/templates/qa/openness_stars.html:24
+#: ckanext/qa/templates/qa/openness_stars_brief.html:23
+#: ckanext/qa/templates/report/openness.html:88
+msgid "Reason"
+msgstr ""
+
+#: ckanext/qa/tasks.py:304
+#, python-format
+msgid "Error details: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:305
+#, python-format
+msgid "Attempted on %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:309 ckanext/qa/tasks.py:317
+#, python-format
+msgid "This URL last worked on: %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:311
+msgid "This was the first attempt."
+msgstr ""
+
+#: ckanext/qa/tasks.py:313
+#, python-format
+msgid "Tried %s times since %s."
+msgstr ""
+
+#: ckanext/qa/tasks.py:319
+msgid "This URL has not worked in the history of this tool."
+msgstr ""
+
+#: ckanext/qa/tasks.py:355 ckanext/qa/tasks.py:386
+msgid "This file had not been downloaded at the time of scoring it."
+msgstr ""
+
+#: ckanext/qa/tasks.py:360
+#, python-format
+msgid "Cache filepath does not exist: \"%s\"."
+msgstr ""
+
+#: ckanext/qa/tasks.py:368
+#, python-format
+msgid ""
+"Content of file appeared to be format \"%s\" which receives openness "
```

Comment:
This looks like a message string not an ID.

Reply:
@ThrawnCA Read above comment, msgid are strings marked for internationalisation.